testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/AbstractKeycloakTest.java

```diff
@@ -96,6 +98,7 @@
  */
 @RunWith(KcArquillian.class)
 @RunAsClient
+@DisableFeature(value = Profile.Feature.ACCOUNT2, skipRestart = true)
```

Comment:
I don't know what happened but my test fixes from https://github.com/ssilvert/keycloak/pull/10 were somehow merged wrong. This line definitely shouldn't be here... @ssilvert Did you do some rebase of my changes or something?

Could you please fix it and re-run both pipelines?